website/docs/docs/use-dbt-semantic-layer/avail-sl-integrations.md

```diff
@@ -9,9 +9,6 @@ meta:
   api_name: dbt Semantic Layer APIs
 ---
 
-<VersionBlock firstVersion="1.6">
```

Comment:
I don't know exactly how this works, but wouldn't removing this make it so our available integrations to the new SL appear before version 1.6 of the docs? We still don't support version < 1.6 of dbt Core, so I think we still want to keep this absent when users are browsing the version of docs that are < 1.6.

Reply:
hey @DevonFulcher ! that's definitely a great point and i intentionally removed it because the prerequisites do let users know they should be on dbt v1.6 and higher to use the SL or metricflow. also, is it a bad thing to have the revamped SL presented to users who are not on dbt v1.6 or higher? it might be a helpful motivator to upgrade their version? 

i do think the initial PR didn't include a callout letting ppl know about the deprecation but i just added it so that users know the legacy SL and dbt metrics was removed and if they'd like to use the SL or metricflow, they can upgrade if they choose -- what do you think about it?

![Screenshot 2023-12-12 at 07 44 45](https://github.com/dbt-labs/docs.getdbt.com/assets/89008547/ee7625b0-f7b8-4cad-9e4f-42617e9ee470)

Reply:
Ya, that sounds great! Thank you!